mc2skos/mc2skos.py

```diff
@@ -296,9 +295,11 @@ def main():
         'include_webdewey': args.webdewey,
         'skip_classification': args.skip_classification,
         'skip_authority': args.skip_authority,
+        'expand': args.expand,
+        'skosify': args.skosify,
     }
 
-    process_records(get_records(in_file), graph, options)
+    graph = process_records(marc.records(), graph, **options)
```

Comment:
I realize I was the one who added a return value to `process_records`, but I'm not totally sure if it makes sense to have it there since the graph is modified in-place..  Feel free to remove it if you like.. or keep it. I'm not totally sure what's best practice for in-place modifying functions.

Reply:
I better remove the return value

Reply:
The return value is needed to get a new `Graph` instance of none is supplied:

    graph = process_records(records, **options)